app/build.gradle

```diff
@@ -16,17 +16,18 @@ repositories {
 //}
 
 ext {
-    androidMapsUtilsVersion = '0.3.4'
     assertjVersion = '3.9.0'
     butterknifeVersion = '8.8.1'
-    constraintLayoutVersion = '1.1.0'
+    constraintLayoutVersion = '1.1.2'
     daggerVersion = '2.13'
     gsonVersion = '2.8.1'
     guavaVersion = '20.0'
     jsonVersion = '20170516'
     junitVersion = '4.12'
     materialDialogsVersion = '0.9.6.0' // Depends on the android support lib. Their version must match ours.
     multidexVersion = '1.0.3'
+    osmbonuspackVersion = '6.5.1'
+    osmdroidVersion = '6.0.1'
```

Comment:
Update to 6.0.2 to hopefully incorporate fix for https://github.com/osmdroid/osmdroid/issues/1055

Reply:
Done.